src/pkg/packaging/rpm/scripts/after_remove_host.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+echo "Removing dotnet host symbolic link"
+sudo unlink /usr/bin/dotnet
```

Comment:
I tested the package without sudo and #!/bin/sh and it worked. I will update for these.